openmrs/src/test/java/org/motechproject/openmrs/tasks/OpenMRSTaskDataProviderTest.java

```diff
@@ -365,9 +365,7 @@ public void shouldReturnNotEnrolledWhenWrongLookupNameForProgramEnrollment() {
 
         ProgramEnrollmentListResult actual = (ProgramEnrollmentListResult) object;
 
-        assertFalse(actual.getResults().get(0).isEnrolled());
-        assertEquals(ProgramEnrollment.NOT_ENROLLED, actual.getResults().get(0).getEnrolledString());
-        assertNull(actual.getResults().get(0).getCurrentState());
+        assertEquals(0, actual.getResults().size());
```

Comment:
You can check if list of results is empty and our property ('numberOfPrograms') returns 0.